src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -180,7 +180,7 @@ static SyntaxNodeExtensions()
             s_bannerMatcher =
                 Matcher.Sequence(
                     Matcher.OneOrMore(commentLine),
-                    s_oneOrMoreBlankLines);
+                    Matcher.Repeat(singleBlankLine));
```

Comment:
Also, there now incorrect comments in GetNodeWithoutLeadingBannerAndPreprocessorDirectives.  It seems like this change is too aggressive.  Won't it treat every comment above a node as a banner?  Do we want to only treat the comments at the very beginning of the file specially?